src/unparsed-blocks/Context.php

```diff
@@ -110,6 +107,18 @@ public function getFilePath(): ?string {
   }
 
   private bool $isHtmlEnabled = false;
+  protected bool $isAllURIEnabled = false;
+  private keyset<string> $allowedURIs = keyset[];
+
+  public function enableAllURISchemes_UNSAFE(): this {
+    $this->isAllURIEnabled = true;
+    return $this;
+  }
+
+  public function enableURISchemeAllowlist(keyset<string> $allowlist): this {
```

Comment:
Same comment as in context.
This method becomes a noop in all uri schemes mode.
How could we improve this API?